test_opensearchpy/test_server/test_rest_api_spec.py

```diff
@@ -74,19 +74,19 @@
     # Warning about date_histogram.interval deprecation is raised randomly
     "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/search_pipeline/10_basic",
     "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/pit/10_basic",
-    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/indices/clone/40_wait_for_completion[0]",
-    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/indices/forcemerge/20_wait_for_completion[0]",
-    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/indices/open/30_wait_for_completion[0]",
-    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/indices/shrink/50_wait_for_completion[0]",
-    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/indices/split/40_wait_for_completion[0]",
+    "OpenSearch-main/rest-api-spec/src/main/resources/rest-api-spec/test/indices/clone/40_wait_for_completion[0]",  # pylint: disable=line-too-long
```

Comment:
Let's increase the allowed line length across the project?

Reply:
what's a good number? 175? that's intellij's default i think.

Reply:
seems like adding it to the command line params and removing from the setup.cfg worked. I'm suspicious...